pkg/server/defaults.go

```diff
@@ -20,6 +20,7 @@ const defaultRegistryUrl = "https://download.daytona.io/daytona"
 const defaultServerDownloadUrl = "https://download.daytona.io/daytona/install.sh"
 const defaultHeadscalePort = 3987
 const defaultApiPort = 3986
+const defaultRegistryPort = 5000
```

Comment:
I suggest we default to `3988` because `5000` might be a busy port a lot of the time.